docs/reference/migration/migrate_8_0/breaker.asciidoc

```diff
@@ -0,0 +1,11 @@
+[float]
+[[breaking_80_breaker_changes]]
+=== Packaging changes
+
+//tag::notable-breaking-changes[]
+[float]
+==== Request Circuit Breaker
```

Comment:
This should say `In flight requests circuit breaker`. We have a *different* circuit breaker that is called "Request circuit breaker" so this is misleading.